spec/integration/group_show_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
```

Comment:
- [ ] It would be better if we add instructions about how to create and migrate the database in the README file. Since the app can not start locally without running the migrations. I'm unable to run the app locally
![Screenshot from 2022-03-24 16-40-07](https://user-images.githubusercontent.com/46388113/159967333-744ca99f-b388-42d3-b73a-810a92cdb700.png)
Kindly pay attention to the migration files